queue/src/fiskaltrust.Middleware.Localization.QueueIT/SignProcessor.cs

```diff
@@ -0,0 +1,97 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using fiskaltrust.ifPOS.v1;
+using fiskaltrust.Middleware.Contracts.Extensions;
+using fiskaltrust.Middleware.Contracts.Interfaces;
+using fiskaltrust.Middleware.Localization.QueueIT.Constants;
+using fiskaltrust.Middleware.Localization.QueueIT.Extensions;
+using fiskaltrust.storage.V0;
+
+namespace fiskaltrust.Middleware.Localization.QueueIT
+{
+    public class SignProcessor : IMarketSpecificSignProcessor
```

Comment:
Any reason for creating two additional `SignProcessors` in the QueueIT, in addition to the already existing one in the general Queue implementation?

Reply:
Yep we'll be able to move this to the Queue package in the next steps. This class is meant to include all "shared" aspects. Still pretty much a WIP though :/